news/20200707.bugfix

```diff
@@ -0,0 +1 @@
+Remove `assert-news` from pre-commit checks
```

Comment:
Is this supposed to have a newline after it or not?

Reply:
asset-news expect only one line per newsfile. If there is more than one line then it'll fail.